lib/abilities.ex

```diff
@@ -9,5 +9,15 @@ defimpl Canada.Can, for: Pairmotron.User do
   def can?(%User{id: user_id}, action, %Group{owner_id: user_id})
     when action in [:edit, :update, :delete], do: true
 
+  def can?(%User{id: user_id}, action, project = %Project{})
+    when action in [:edit, :update, :delete] do
+    if is_nil(project.group_id) do
+      false
+    else
+      project = project |> Pairmotron.Repo.preload(:group)
+      project.group.owner_id == user_id
+    end
+  end
```

Comment:
Only the owner of a group can edit a project associated with that group. Eventually this will be expanded to include users with the relevant GroupRoles, but for now owners are the only ones authorized.